wrappers/python/CMakeLists.txt

```diff
@@ -7,6 +7,17 @@ if (NOT BUILD_PYTHON_BINDINGS)
     message(WARNING "Python Bindings being built despite unset option because they are required for python documentation")
 endif()
 
+if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.12)
```

Comment:
@morxa , I moved the testing to the Debina builds but still - the follow statement is not compatible with Cmake v3.5 on Ubuntu 16 and produces the following error

```sh
16:49:53 CMake Error at wrappers/python/CMakeLists.txt:10 (if):
16:49:53   if given arguments:
16:49:53 
16:49:53     "CMAKE_VERSION" "VERSION_GREATER_EQUAL" "3.12"
16:49:53 
16:49:53   Unknown arguments specified
16:49:53 
16:49:53 
16:49:53 -- Configuring incomplete, errors occurred!
```

Invoked with the following configuration (python 3.5)
16:49:52 cmake config:../../ -G "Unix Makefiles" -DPYTHON_EXECUTABLE=/usr/bin/python3.5 -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_EXAMPLES=true -DBUILD_SHARED_LIBS=false -DBUILD_WITH_OPENMP=false -DENFORCE_METADATA=true -DBUILD_CSHARP_BINDINGS=false -DBUILD_PYTHON_BINDINGS=true -DBUILD_PYTHON_DOCS=false -DBUILD_WITH_TM2=true -DCMAKE_INSTALL_PREFIX=../../realsense2

Reply:
Older cmake does not know `VERSION_GREATER_EQUAL`, only `VERSION_GREATER`. :/

I inversed the logic, can you check again? Thanks!

Reply:
I'll try it again

Reply:
@morxa ,please rebase it with v2.35.2 or later - the gaps are too big to bridge...

Reply:
Done!